.github/workflows/protobuf.yml

```diff
@@ -33,3 +33,30 @@ jobs:
         with:
           input: "proto"
           against: "https://github.com/${{ github.repository }}.git#branch=${{ github.event.pull_request.base.ref }},ref=HEAD~1,subdir=proto"
+
+  proto-gen-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.23"
+      - name: Install proto dependencies
+        run: make proto-deps
```

Comment:
Oh wow this was it! I ran `make proto-deps` and regenerated the protos and the diff went away. 

So my local machine's proto deps were different than what was expected.